### PR TITLE
PHP-144 - Persistent sessions reference counting issue

### PR DESCRIPTION
**Pull request by mpenick**
Fixed issue where persistent sessions were incorrectly double decrementing the
reference count; Once when deleting from the hash and once in the persistent
session dtor.

### CODE REVIEW

File: ext/src/DefaultCluster.c [hunk 1]
```diff
@@ -117,11 +117,7 @@ PHP_METHOD(DefaultCluster, connect)
 
   if (php_driver_future_is_error(future TSRMLS_CC) == FAILURE) {
     if (session->persist) {
-      if (PHP5TO7_ZEND_HASH_DEL(&EG(persistent_list), hash_key, hash_key_len + 1)) {
```
Comment (jippi) [inline ext/src/DefaultCluster.c:120]: `// done, no it was not :P`

Comment (mpenick) [reply ext/src/DefaultCluster.c:120]: heh, that's exactly what I was thinking as I was adding this...

**Comment by stamhankar999** [comment]
👍

**Comment by mpenick** [comment]
Merged at f1300a21af7165305ab983a1e9c9c46be3d0ee6f

**Pull request closed without merging**